packages/core/src/Stores/notification-store.js

```diff
@@ -335,6 +335,16 @@ export default class NotificationStore extends BaseStore {
     @action.bound
     init() {
         this.setClientNotifications();
+        reaction(
+            () => [
+                this.root_store.client.is_uk,
+                this.root_store.client.has_malta_account,
+                this.root_store.client.can_have_mlt_account,
+            ],
+            () => {
+                this.setClientNotifications();
```

Comment:
reverted previously used init() to set notifications while other stores are being initialized (in app.js) and set a reaction to reset notfications when the listed dependencies get updated.